app/controllers/dashboard_controller.rb

```diff
@@ -2,6 +2,7 @@ class DashboardController < ApplicationController
   skip_before_action :authenticate_user!, :only => :help
 
   layout 'help', :only => :help
+  layout "tailwind"
```

Comment:
i think we need to move this into the index action only, rather than all actions in dashboard_controller